### ISSUE TITLE
PvP Unit Frames (Name Plates)

### ISSUE THREAD

**Issue by HoneyBearNathan** [GoldpawsStuff/AzeriteUI#66]
**What WoW version do you wish to see this feature in?**
- [ ] Classic
- [x] Battle for Azeroth

**Is your feature request related to a problem? Please describe.**
I enjoy casually playing PvP but I dislike not seeing my enemies class before an arena game because it puts me in a disadvantage since I can't plan ahead like the default WoW UI would allow me to.
I also dislike not seeing battleground flag carriers appear on my UI since with default, I can click on the opposite faction's flag carrier and just see his name plate anywhere I am, which allows me to attack him instead of me just guessing.

**Describe the solution you'd like**
I'd like for you to implement name plates for arena. (you could style them to appear as boss name plates on the side of the screen, where the name is instead their specialization and their health bar of course can be class coloured)
I'd like to also see this applied to flag carriers in battlegrounds (Name is instead spec/class colour health bar)
(Note: I don't really care about their cooldowns or anything extra like that, I'd just really enjoy the name plates on the side of screen in your UI style)

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered. You can list other addons you're using to achieve this here.
I'd resorted to using other addons for the arena issue (Gladius) but I found it clunky, annoying to use, and just not visually appealing at all
I haven't found any alternatives for my battleground flag carrier issue.

**Additional context**

**Comment by HoneyBearNathan** [comment]
Sorry for not labeling it correctly, as well as not looking at the roadmap. I'm not entirely sure how this site works just yet.

**Comment by Mithgul** [comment]
Exactly! I really like a lot this addon. But only thing i complain is that i cant see party frames(with portrait) in arena...I'm enjoying with healer layout in dungeons and got some weak auras with this ui. But in arena i see them like raid frames. Can we please use party frames in arenas like we use in dungeons?
